src/keyszer/transform.py

```diff
@@ -216,6 +216,31 @@ def dump_diagnostics():
     print("")
 
 
+# ─── COMBO CONTEXT LOGGING ────────────────────────────────────────────────────────
+
+
+def log_combo_context(combo, ctx, keymap, _active_keymaps):
+    """Log context around usage of combo"""
+    import textwrap
+
+    debug("")
+    debug(f"WM_CLS: '{ctx.wm_class}' | WM_NME: '{ctx.wm_name}'")
+    debug(f"DVN: '{ctx.device_name}' | CLK: '{ctx.capslock_on}' | NLK: '{ctx.numlock_on}'")
+    debug(f'ACTIVE KEYMAPS:')
+
+    indent = ' ' * 5
+    max_len = max(80 - len(indent), 64)
+    wrapped_items = textwrap.wrap(", ".join([f"'{item.name}'" for item in _active_keymaps]), width=max_len)
+    output_str = f"{indent}{wrapped_items[0]}"
+    for item in wrapped_items[1:]:
```

Comment:
I feel this is misguided... if something wraps weirdly you'd want that to *stand out*, not be hidden.  So I don't think any of this extra complexity is needed...

Reply:
> I feel this is misguided... if something wraps weirdly you'd want that to _stand out_, not be hidden. So I don't think any of this extra complexity is needed...

I actually have no idea what you mean by this. Wraps weirdly? What’s being hidden?

Reply:
If there's no wrapping algorithm at all, the terminal will even cut words apart, after the output reaches the right edge of the window. The wrapper at least cuts at word boundaries. 

Not sure what you want done here, or what you're objecting to.

Reply:
I can add some ellipses characters to connect the wrapped items.  

```sh
(DD) ACTIVE KEYMAPS:
     'User hardware keys', 'Wordwise - not vscode', 'GenTerms overrides:
      … Ubuntu/Fedora', 'GenTerms overrides: GNOME', 'General Terminals', 'GenGUI
      … overrides: not Chromebook', 'GenGUI overrides: Fedora', 'GenGUI overrides:
      … GNOME', 'General GUI'
```

Reply:
That's. better idea.

Reply:
OK, it's in there. Anything else?